### PR TITLE
Parametrize removal based on occurrence count

### PR DESCRIPTION
**Pull request by fridex**

### CODE REVIEW

File: f8a_tagger/recipes.py [hunk 1]
```diff
@@ -219,10 +221,10 @@ def aggregate(input_keywords_file, no_synonyms=None, use_progressbar=False):  #
                     all_keywords[str(keyword)]['synonyms'] = synonyms
 
     # filter out keywords with low occurrence count
-    if defaults.OCCURRENCE_COUNT_FILTER > 1:
+    if occurrence_count_filter > 1:
         result = {}
         for keyword, value in all_keywords.items():
-            if value.get('occurrence_count', 1) > defaults.OCCURRENCE_COUNT_FILTER:
+            if value.get('occurrence_count', 1) > occurrence_count_filter:
```
Comment (sara-02) [inline f8a_tagger/recipes.py]: Min value should be zero? Or it is kept 1 on purpose

Comment (fridex) [inline f8a_tagger/recipes.py]: It's default if occurrence count is not present - assuming that if a keyword is registered, it had to be used somewhere - thus 1.

Comment (sara-02) [inline f8a_tagger/recipes.py]: @fridex makes sense now.

**Comment by sara-02** [comment]
Rest looks good.

**Pull request merged**